### PR TITLE
Rename airbyte-config module

### PR DESCRIPTION
**Pull request by rodireich**
## What
The module airbyte-config was transferred to airbytehq/platform but also remained in airbytehq/airbyte, used mainly by the connector test harness.
The duplicate name creates a confusing situation as platform modules are also imported into airbyte projects.
The goal of this rename is to create distinction between platform Airbyte-config and Airbyte's.

## How
append `oss` to the module and packages inside airbytehq/airbyte

airbyte
    |
    ––– airbyte-config → airbyte-config-oss
                     |
                     ––– config-models → config-models-oss
                     |
                     ––– init → init-oss
                     |
                     ––– specs → specs-oss

### CODE REVIEW

**Comment by rodireich** [comment]
/connector-performance connector=connectors/source-postgres ref=rodi/rename-module-airbyte-config

**Review by c-p-b — APPROVED** [review]
Thanks Rodi. I think this will help a lot to eliminate confusion

File: airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg
Comment (evantahler) [inline airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: This one is probably a rename on master

**Review by evantahler — COMMENTED** [review]
With the icon renames, are there any changes needed to the platform for the front-end?

File: airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg
Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: > This one is probably a rename on master
Do you mean there's a change needed on frontend?

Comment (evantahler) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: I think you just need to pull master for this one https://github.com/airbytehq/airbyte/pull/24821

Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: Got it. I'm not quite sure why It's not on this branch as I merged from master today.
checking

Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: This [branch](https://github.com/airbytehq/airbyte/commits/rodi/rename-module-airbyte-config) has your commit from 4/3.
I don't get why icon didn't retain the updated aws-datalake name 🤔

Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: I manually reverted name to aws-datalake
I'm not sure why this had happened

**Comment by rodireich** [comment]
> I'm not super familiar with the `airbyte-config` module, given that it currently resides in two different repos is there a reason why one repo isn't the source of truth and versioned by publishing some `jar` and we can remove the version within this repo to point to that instead?
> 
> cc: @cpdeethree

I had the same thought.
See this [thread](https://airbytehq-team.slack.com/archives/C02U2SSHP9S/p1678990946237829)

File: airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg
Comment (rodireich) [reply airbyte-config-oss/init-oss/src/main/resources/icons/awsdatalake.svg:1]: @evantahler can you please confirm whether this is ok with you?

**Comment by rodireich** [comment]
/connector-performance connector=connectors/source-postgres ref=rodi/rename-module-airbyte-config

**Pull request merged**